pkg/config/config.go

```diff
@@ -937,12 +940,18 @@ func (c *Config) Write() error {
 	return nil
 }
 
-// Reload reloads the configuration from containers.conf files
+// InvalidateCache invalidates the config cache.  This function is meant to be
+// used for long-running processes that need to reload potential changes made to
+// the cached containers.conf files.
+func InvalidateCache() {
```

Comment:
Can we add an `error` return value? Even though we always return nil at the moment, we keep the door open for future changes without breaking the API. I do that from time to time when I am not sure how the API will evolve.

Reply:
Make this private.